src/aliases/store/migrations/001_add_aliases.up.sql

```diff
@@ -0,0 +1,12 @@
+begin;
+
+create table if not exists aliases (
+	id serial primary key,
+	key text,
+	registry_name text,
+	value json
+);
+create unique index on aliases (registry_name, key);
+create index on aliases (registry_name);
```

Comment:
This index won't speed up your queries. Since `registry_name` is the first part of the other unique index. If you want I can explain more in detail what is that.

Reply:
got it.